libraries/lwc/meta/summary.md

```diff
@@ -0,0 +1,14 @@
+<h4 id="lwc-handling-data">Handling data</h4>
+
+When passing data to a custom element, LWC takes a properties-if-available approach where attributes
+are set by default, but properties are set when they exist. This heuristical approach involves a
+runtime check to see whether a property is defined, and as such, data will be passed as an attribute
+if the custom element has not been upgraded. It is the responsibility of the component author to
+handle this scenario.
+
+<h4 id="lwc-handling-events">Handling events</h4>
+
+LWC has no restrictions on event names when listening for events imperatively via the
```

Comment:
No library has restrictions on `addEventListener()` since it's a native method. Do other libraries/frameworks mention this this way?